lib/core/presentation/pages/notification/notifications_listing_page.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:app/core/presentation/widgets/lemon_appbar_widget.dart';
+import 'package:app/core/presentation/widgets/loading_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/service/notification/notification_service.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:app/core/utils/navigation_utils.dart';
+
+@RoutePage()
+class NotificationPage extends StatelessWidget {
+  late final NotificationService notificationService = NotificationService(NotificationRepositoryImpl());
+
+  Widget _notificationsListingBlocProvider(Widget child) {
+    return BlocProvider<NotificationsListingBloc>(
+      create: (context) => NotificationsListingBloc(notificationService)..add(NotificationsListingEvent.fetch()),
+      child: child,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return _notificationsListingBlocProvider(
+      _NotificationsListingView(),
+    );
+  }
+}
+
+class _NotificationsListingView extends StatefulWidget {
+  const _NotificationsListingView();
+
+  @override
+  State<_NotificationsListingView> createState() => _NotificationsListingViewState();
+}
+
+class _NotificationsListingViewState extends State<_NotificationsListingView> {
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final themeColor = Theme.of(context).colorScheme;
+    return Scaffold(
+      appBar: LemonAppBar(
+        title: t.notification.notifications,
+        leading: BurgerMenu(),
+        actions: [
+          ThemeSvgIcon(color: themeColor.onSurface, builder: (filter) => Assets.icons.icChat.svg(colorFilter: filter)),
+        ],
+      ),
+      backgroundColor: themeColor.primary,
+      body: BlocBuilder<NotificationsListingBloc, NotificationsListingState>(
+        builder: (context, state) {
+          return state.when(
+            loading: () => Loading.defaultLoading(context),
+            fetched: (filteredNotifications) {
+              return ListView.separated(
+                itemBuilder: (ctx, index) => index == filteredNotifications.length
+                    ? const SizedBox(height: 80)
+                    : NotificationCard(
+                        key: Key(filteredNotifications[index].id ?? ''),
+                        notification: filteredNotifications[index],
+                        onTap: () {
+                          NavigationUtils.handleNotificationNavigate(context, filteredNotifications[index]);
+                        },
+                      ),
+                separatorBuilder: (ctx, index) =>
+                    SizedBox(height: Spacing.extraSmall),
+                itemCount: filteredNotifications.length + 1,
```

Comment:
I think we won't need to this anymore, because I have moved the bottom app bar into the scaffold, so now the bottom tabbar should not hide the list tail anymore

Reply:
ah I mean we no need to +1 any more @sieuhuflit